.github/workflows/update_gh_pages.yml

```diff
@@ -1,8 +1,8 @@
 name: Update gh-pages
 on:
-  # Trigger on UTC noon daily, or manually.
+  # Trigger on every three hours, or manually.
   schedule:
-    - cron: '0 12 * * *'
+    - cron: '20 /3 * * *'
```

Comment:
I'm not an expert on cron syntax, but is there a meaningful difference between `20 /3 * * *` and `20 */3 * * *`? crontab.guru is happier with [`*/3`](https://crontab.guru/#20_*/3_*_*_*) vs [`/3`](https://crontab.guru/#0_/3_*_*_*)

Reply:
Yes, the meaningful difference is I don't know what I'm doing and only `*/3` is valid.